src/Rings/groebner.jl

```diff
@@ -311,3 +311,121 @@ function leading_ideal(I::MPolyIdeal, ord::Symbol)
   return leading_ideal(groebner_basis(I, ord), ord)
 end
 
+@doc Markdown.doc"""
+    normal_form_internal(I::Singular.sideal, J::MPolyIdeal)
+
+**Note**: Internal function, subject to change, do not use.
+
+Compute the normal form of the generators of the ideal `I` w.r.t. a
```

Comment:
Which generators? `gens(I)`? I think we should try to spell it out.

Reply:
Alright, added `gens(I)` to the docu description.